src/components/shared/RightSidebar/RightSidebar.tsx

```diff
@@ -0,0 +1,26 @@
+// "use client";
+
+// import { usePathname } from "next/navigation";
+import React from "react";
+import ProfileLink from "./ProfileLink";
+import TagsList from "./TagsList";
+import SocialLinks from "./SocialLinks";
+import { useRouter } from "next/router";
+// import { useRouter } from "next/navigation";
+
+const RightSidebar = () => {
+  // const pathname = usePathname();
+  // const router = useRouter();
+  // const pathname = router.pathname;
+  return (
+    <aside className="flex min-h-screen w-3/12 flex-col gap-2 bg-myBlack-800 p-4">
+      <ProfileLink />
+      {/* {!pathname.includes("profile") ? <TagsList /> : <SocialLinks />} */}
+      {/* <TagsList /> */}
+      <TagsList />
+      <SocialLinks />
+    </aside>
+  );
+};
+
+export default RightSidebar;
```

Comment:
```suggestion

import React from "react";
import ProfileLink from "./ProfileLink";
import TagsList from "./TagsList";
import SocialLinks from "./SocialLinks";

const RightSidebar = () => {
  return (
    <aside className="flex min-h-screen w-3/12 flex-col gap-2 bg-myBlack-800 p-4">
      <ProfileLink />
      <TagsList />
      <SocialLinks />
    </aside>
  );
};

export default RightSidebar;
```
Since we're using the Social Links and the TagList to determine what's rendered, these commented out sections can be removed.